Source/bw6-maven-plugin/src/main/java/com/tibco/bw/maven/plugin/module/BWModulePackageMojo.java

```diff
@@ -183,20 +183,21 @@ private void addDependencies() {
 				continue;
 			}
 			
-			
+			boolean isSharedModule = false;
 			Manifest mf = ManifestParser.parseManifestFromJAR( file);
 			for( Object str : mf.getMainAttributes().keySet())
 			{
 				getLog().debug( str.toString() );
-				if( "TIBCO-BW-SharedModule".equals(str.toString() ))
-				{
-					continue;
-					
+				if("TIBCO-BW-SharedModule".equals(str.toString())) {
```

Comment:
Recommend to use Constants.TIBCO_SHARED_MODULE instead of plain String.

Reply:
I will fix it.